src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -771,6 +772,9 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
 * *_Shared Image Gallery:_* The prefix `gallery#` allows you to use images from an existing Azure https://docs.microsoft.com/en-us/azure/virtual-machines/linux/shared-image-galleries[Shared Image Gallery]. Append the name of your Resource Group, your Gallery and your Image as `resourceGroup:gallery:image`. Optionally, you can specify a version too (`:version`); if no version is given, the latest image version will be used by ProActive. For example: +
 `gallery#proactive-rg:activeeon-gallery:my-image` +
 `gallery#myRG:myGallery:myImage:1.0`
+** *_Shared Image Gallery from non-default subscription_* It is possible to use images from an existing Azure https://docs.microsoft.com/en-us/azure/virtual-machines/linux/shared-image-galleries[Shared Image Gallery] from the specific (different from default) subscription. +
+The non-default subscription should be accessible with your Azure Credentials provided in the *azureCredentialFile* configuration. +
+In case of retrieving an image from the specific subscription, the prefix `sub#` should be added and the _subscriptionId_ should be provided in the next format: `gallery#sub#subsriptionId:resourceGroup:gallery:image[:version]`
```

Comment:
Maybe it would make sens to have this sentence switch with the previous one to ease the understanding. In this case, `In case of retrieving an image from the specific subscription` should be removed.